float32 linear_velocity_x
float32 linear_velocity_y
float32 angular_velocity_z
float32 angular_velocity_head_tilt
float32 angular_velocity_head_pan
